version/news/NEWS-2024.04.0-rel-chocolate-cosmos.md

```diff
@@ -147,3 +147,6 @@
 - Updated nginx to 1.25.4 (rstudio-pro#5970)
 - Updated node.js to version 18.19.1 (#14373, rstudio-pro#5935)
 - Updated zlib to 1.3.1 (rstudio-pro#5971)
+
+### Deprecated / Removed
+- Due to the upcoming [end of support for CentOS 7](https://posit.co/about/platform-support/) at the end of June 2024, CentOS 7 builds are no longer available for public download. If you have any questions please contact Posit Support.
```

Comment:
```suggestion
- Due to the upcoming [end of support for CentOS 7](https://posit.co/about/platform-support/) in June 2024, CentOS 7 builds are no longer available for public download. If you have any questions, please contact Posit Support.
```

Not sure if you want to add a link to the support page in the admin guide but here is the link if you do: https://docs.posit.co/ide/server-pro/getting_started/support.html